deny.toml

```diff
@@ -7,7 +7,30 @@ allow = [
     "Apache-2.0",
     "ISC",
     "BSD-3-Clause",
-    "OpenSSL"
+]
+
+exceptions = [
+    { allow = ["MPL-2.0"], crate = "webpki-roots" }, # This crate is a dependency of `reqwest`.
```

Comment:
Need to check if these licenses are compatible with our licensing model.

Reply:
I recall seeing it somewhere in CNCF exception list - but good to check.

Reply:
These two are not listed under the allow list: https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md#approved-licenses-for-allowlist

Reply:
Ok, I think I saw some components with MPL-2 license in the exception list, like this - https://github.com/cncf/foundation/blob/main/license-exceptions/cncf-exceptions-2023-08-31.json

btw, the latest version of webpki-roots seems to be Apache licensed - https://github.com/rustls/webpki-roots/tree/v/0.26.8

Reply:
Also, we don't expose any of these crates' (API or data) through this repo's components, so we should be good.